### ISSUE TITLE
0.10.5 breaks installation

### ISSUE THREAD

**Issue by antonnyman** [GetStream/stream-video-js#1470]
**Which package/packages do you use?**

- [ ] `@stream-io/video-react-sdk`
- [x] `@stream-io/video-react-native-sdk`
- [ ] `@stream-io/video-client`

**Describe the bug**
Fresh install of the SDK v0.10.5 breaks with error:

```sh
❌ /[...]/node_modules/@stream-io/video-react-native-sdk/ios/RTCViewPipManager.swift:25:41: value of type 'RCTBridge' has no member 'uiManager'
            let component = self.bridge.uiManager.view(
```

**To Reproduce**

1. `npm/yarn install @stream-io/video-react-native-sdk@0.10.5`
2. `cd ios && pod install`

**Expected behavior**
Installation should complete.

**Version**

0.10.5

**React Native**

macOS 14.6.1
React Native 0.72.11

**Node JS client**

- 20.11.1

**Comment by santhoshvai** [comment]
Hi @antonnyman, could you please do the following in your podfile

Open the file `./ios/Podfile` and add this line inside your targets (right before the `use_react_native` line in current react-native releases that calls the react native Podfile function to get the native modules config):

`use_frameworks! :linkage => :static`

 -- Apologies for the rouble here, we ll add it to the docs very soon with an explanation of why

**Comment by oliverlaz** [comment]
hi @antonnyman, we just released v0.10.6 which ships a fix for this issue. 
Can you please check if everything looks good on your side after upgrading to it?
